adal/src/main/java/com/microsoft/aad/adal/AuthenticationSettings.java

```diff
@@ -253,7 +253,7 @@ public int getConnectTimeOut() {
      * timeout elapses before a connection is established. Default value is
      * 30000 milliseconds.
      * 
-     * @param timeOutMillis the connect timeout in milliseconds. Non-negative
+     * @param timeOutMillis the connect timeout in milliseconds. Non-negative.
```

Comment:
nit: usually we do @param timeOutMillis the non-negative connect time out in milliseconds.